test/spec/cloud-element-templates/cmd/ChangeElementTemplateHandler.spec.js

```diff
@@ -2158,6 +2158,91 @@ describe('cloud-element-templates/cmd - ChangeElementTemplateHandler', function(
 
     });
 
+    describe('zeebe:formDefinition', function() {
+      beforeEach(bootstrap(require('./form-definition.bpmn').default));
+
+      const newTemplate = require('./form-definition.json');
+
+      it('should execute', inject(function(elementRegistry) {
+
+        // given
+        let task = elementRegistry.get('Camunda_user_Task_no_implementation');
+
+        // when
+        changeTemplate(task, newTemplate);
+
+        // then
+        task = elementRegistry.get('Camunda_user_Task_no_implementation');
+        expectElementTemplate(task, 'form-definition-template');
+
+        const formDefinition = findExtension(task, 'zeebe:FormDefinition');
+
+        expect(formDefinition).to.exist;
+        expect(formDefinition).to.have.property('formId', 'complexFormId');
+      }));
+
+      it('undo', inject(function(commandStack, elementRegistry) {
+
+        // given
+        let task = elementRegistry.get('Camunda_user_Task_no_implementation');
+
+        changeTemplate(task, newTemplate);
+
+        // when
+        commandStack.undo();
+
+        // then
+        task = elementRegistry.get('Camunda_user_Task_no_implementation');
+        expectNoElementTemplate(task);
+
+        const formDefinition = findExtension(task, 'zeebe:FormDefinition');
+
+        expect(formDefinition).not.to.exist;
+
+      }));
+
+
+      it('redo', inject(function(commandStack, elementRegistry) {
+
+        // given
+        let task = elementRegistry.get('Camunda_user_Task_no_implementation');
+
+        changeTemplate(task, newTemplate);
+
+        // when
+        commandStack.undo();
+        commandStack.redo();
+
+        // then
+        task = elementRegistry.get('Camunda_user_Task_no_implementation');
+        expectElementTemplate(task, 'form-definition-template');
+
+        const formDefinition = findExtension(task, 'zeebe:FormDefinition');
+
+        expect(formDefinition).to.exist;
+        expect(formDefinition).to.have.property('formId', 'complexFormId');
+      }));
+
+      it('should discard', inject(function(elementRegistry) {
+
+        // given
+        const task = elementRegistry.get('Job_worker_user_task_form_key');
+
+        // when
+        changeTemplate(task, newTemplate);
+
+        // then
+        expectElementTemplate(task, 'form-definition-template');
+
+        const formDefinition = findExtension(task, 'zeebe:FormDefinition');
+
+        expect(formDefinition).to.exist;
+
+        expect(formDefinition).to.have.property('formId', 'complexFormId');
+        expect(formDefinition).to.not.have.property('formKey', 'formKey');
```

Comment:
@yT0n1  add test "should not override existing" ; https://github.com/bpmn-io/bpmn-js-element-templates/pull/155/files?diff=unified&w=1#:~:text=it(%27should%20not%20override%20existing%27%2C%20inject(function(elementRegistry)%20%7B